doc/rst/language/spec/classes.rst

```diff
@@ -123,9 +123,6 @@ Additionally, coercions are available that are equivalent to calling the
         var bc: borrowed C = own;  // 'bc' stores the result of own.borrow()
                                    // due to coercion from owned C to C
```

Comment:
For the same reason as the below change, should this comment say "from owned C to borrowed C"?

Reply:
yes, thanks

Reply:
PR #15141 will do it.